tests/Cargo.toml

```diff
@@ -60,3 +60,4 @@ path = "src/tests/da.rs"
 
 [features]
 metrics = ["nomos-node/metrics"]
+debug = []
```

Comment:
Probably we should add a default feature with nothing activated? 🤔

Reply:
Is it really necessary? Not sure what the benefit would be, but maybe it's a convention?

Reply:
I'm not sure actually. Usually would be to activate other features by default. So probably for empty case is not necessary. Up to you.

Reply:
Lets add it when we'll need it, I'll re-request the review, minor change for clippy.